strictdoc/core/document_iterator.py

```diff
@@ -9,6 +9,19 @@
 )
 from strictdoc.backend.sdoc.models.section import FreeText, SDocSection
 
+DocumentAny = NewType(
```

Comment:
This will probably not work. Let me handle this myself in a separate PR.

Reply:
Yes, fixed it already locally. Needs to be alias, not newtype. It's an unrelated drive by change anyways. Thanks if you handle it separately.

Reply:
Yes, I also see that you don't really need it for this PR. The change is nice though, so I will handle it.